web/modules/custom/hoeringsportal_citizen_proposal/src/Form/ProposalFormSupport.php

```diff
@@ -118,11 +120,16 @@ public function submitForm(array &$form, FormStateInterface $form_state): void {
       return;
     }
 
-    $this->helper->saveSupport($this->getUserUuid(), $node, [
-      'user_name' => $form_state->getValue('name'),
-      'user_email' => $form_state->getValue('email'),
-      'created' => time(),
-    ]);
+    $this->helper->saveSupport(
+      $this->getUserUuid(),
+      $node,
+      [
+        'user_name' => $form_state->getValue('name'),
+        'user_email' => $form_state->getValue('email'),
+        'created' => time(),
+      ],
+      $adminFormStateValues['support_submission_text']
+    );
```

Comment:
I think it would be more clean to keep the message stuff in the form and let `saveSupport` throw an exception (after logging it) on errors, i.e. something along the lines of

```php
try {
  $this->helper->saveSupport(…);
  $this->messenger->addStatus($adminFormStateValues['support_submission_text']);
} catch (\Exception $e) {
  $this->messenger->addError(…);
}
```